src/pages/selection/capital/index.jsx

```diff
@@ -53,19 +49,19 @@ class Capital extends Component {
           name={mcQuestions.ballotIssueId}
           values={mcQuestions.ballotIssueOptions}
         />
-      )
-    })
+      );
+    });
 
     return (
       <div className='container'>
         <div className='row'>
           <div className='col-md-12'>
             <SectionHeader
-              title={this.state.header.pageTitle} 
-              subtitle=""
+              title={this.state.header.pageTitle}
+              subtitle=''
```

Comment:
We don't need to put `subtitle=''` when we don't have any subtitle. We can remove this line.

Reply:
Fixed!